env.sh

```diff
@@ -76,7 +76,7 @@ test -t 1 && USE_TTY="-t"
 # Condition `daemon` to run daemon.
 if [ "$3" == "--daemon"  ]; then
     set "${@:1:2}" "${@:4}"
-    docker exec -d ${CONTAINER_NAME} /usr/bin/gosu ${USER_NAME} "$@"
+    docker exec -d ${CONTAINER_NAME} /bin/bash -c "/usr/bin/gosu ${USER_NAME} $* >/dev/null 2>&1"
```

Comment:
User got the problem while they are *running CITA*, so `scripts/env.sh` is the exactly file needs to be changed?
And, we have a feature which need to print service's log to STDOUT, is ok for using `>/dev/null`?

Should we need to double check about this issue?

Reply:
As I said in the content of this PR:
> This is a temporary solution, I think we should run a script in docker, instead of passing a shell command.

And,
> `docker exec -it` will hang if there is any daemon process which hold the `stdout`.

So, we have no choice:
if you want to run a daemon process in docker and you want to have ability to use `docker exec -it` to attach into the docker environment, you **HAVE TO** redirect the `stdout`.

Yes, we can redirect `stdout` into a file, but, why not just keep it simple, do it just in the configuration of logs.

Reply:
> we have a feature which need to print service's log to STDOUT
Do not use `cita-env`, then the problem is solved